lib/web_ui/lib/src/engine/canvaskit/embedded_views.dart

```diff
@@ -390,7 +390,12 @@ class HtmlViewEmbedder {
       debugBoundsCanvas ??= rasterizer.displayFactory.getCanvas();
       final CkPictureRecorder boundsRecorder = CkPictureRecorder();
       final CkCanvas boundsCanvas = boundsRecorder.beginRecording(
-          ui.Rect.fromLTWH(0, 0, _frameSize.width, _frameSize.height));
+          ui.Rect.fromLTWH(
+        0,
+        0,
+        _frameSize.width.toDouble(),
+        _frameSize.height.toDouble(),
+      ));
```

Comment:
nit: this is some strange indenting. Maybe add a `,` between the two `))`?

Reply:
done, but it didn't help much unfortunately :\